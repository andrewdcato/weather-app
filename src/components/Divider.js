import React from 'react';

export default function Divider() {
  return (
    <div className="block sm:flex justify-between items-center flex-wrap"></div>
  )
}
