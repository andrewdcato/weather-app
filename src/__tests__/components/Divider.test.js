import React from 'react';
import renderer from 'react-test-renderer';
import Divider from '../../components/Divider';

test('matches snapshot', () => {
  const divider = renderer.create(<Divider />).toJSON();
  expect(divider).toMatchSnapshot();
});
