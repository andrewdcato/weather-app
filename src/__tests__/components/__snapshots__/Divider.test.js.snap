// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="block sm:flex justify-between items-center flex-wrap"
/>
`;
